### DISCUSSION TITLE
怎么训练一个过拟合的网络？

### DISCUSSION THREAD

**Discussion by Gaondong** [open-mmlab/mmdetection#9621] [Q&A] [answered]
我想要使用100张图训练centernet， 只在mmdet centernet resnet18中打开了auto_scale_lr， 数据集repeat20次，20*28epoch，ap50能到50多分，但目标confidence很低，甚至有的目标检测不到

**Comment by Li-Qingyun** [comment, marked as answer]
光打开auto_scale_lr应该是不够的

**Reply by Li-Qingyun** [reply]
auto_scale_lr 里预设的参数一般针对coco数据集的吧   你自己的数据集可能不适用的
还是要调一下lr的：如果你的训练时间比较短，直接挂一个遍历lr的程序，挑一个最好的就行了

而且感觉，你这小数据集，迭代次数也应该是不够。我没训练过你的这么小的过，但是对于稍微小一点的数据集，我的经验还得是尽量多训练过度训练一次实验，可以开一个tensorboard观测训练曲线，观察训练时常和你模型精度的关系，来做个 trade off 的决定以后用什么策略训练

其余的参数也可能需要调一调的~

**Reply by Gaondong** [reply]
> auto_scale_lr 里预设的参数一般针对coco数据集的吧 你自己的数据集可能不适用的 还是要调一下lr的：如果你的训练时间比较短，直接挂一个遍历lr的程序，挑一个最好的就行了
> 
> 而且感觉，你这小数据集，迭代次数也应该是不够。我没训练过你的这么小的过，但是对于稍微小一点的数据集，我的经验还得是尽量多训练过度训练一次实验，可以开一个tensorboard观测训练曲线，观察训练时常和你模型精度的关系，来做个 trade off 的决定以后用什么策略训练
> 
> 其余的参数也可能需要调一调的~

感谢回复~， 已经找到问题了，问题是test pipeline中 resize我配置了训练时的尺寸，但由于MultiScaleFlipAug增广中scale factor没有修改，导致resize根本没起到作用， 修改scale factor后，网络推理基本正常了

**Reply by Li-Qingyun** [reply]
okkkk
